cli/src/main.rs

```diff
@@ -90,10 +93,20 @@ fn client_from_args(args: &Args, config: &ReinferConfig) -> Result<Client> {
         ));
     }
 
+    // Retry everything but the very first request.
+    // Retry wait schedule is [10s, 20s, 40s, fail]. (Plus the time for each attempt to timeout.)
+    let timeout_handling = TimeoutHandling {
+        retry_on_timeout: RetryOnTimeout::Automatic,
+        max_retry_count: 3,
+        base_wait: std::time::Duration::from_secs_f64(10.0),
+        backoff_factor: 2.0,
+    };
```

Comment:
Willing to accept bikeshedding on whether these defaults are good, and also whether they should be configurable via global cmdline arguments (or environment variables).

Reply:
There's a lot of bikeshedding to do on backoff retries, but I'm happy to have this in as is. Maybe I'd default to 5.0s rather than 10, but whatever.